cmd/backup/backup.go

```diff
@@ -77,17 +77,20 @@ func (c *CmdOpts) backup() error {
 		logger.Fatalf("cannot find data-dir (%v). check your environment and/or command input and try again.", c.K0sVars.DataDir)
 	}
 
-	role := install.GetRoleByStagedKubelet(c.K0sVars.BinDir)
-	logrus.Debugf("detected role for backup operations: %v", role)
+	status, err := install.GetStatusInfo(c.K0sVars.RunDir, config.StatusSocket)
+	if err != nil {
+		logger.Fatalf("unable to detect cluster status %s", err)
```

Comment:
let's keep it consistent with whole function (it always return error, instead of doing Fatalf and kill process here).
In general, logger.Fatalf is not the best practice, for example, it prevents "defers" from being called (not this particular function case, just generic observation)

Reply:
good point